File: Semester 4/Advanced Programming/RpcProtocolC#/Model/Rezervare.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain
{
    [Serializable]
    public class Rezervare : Entity<int>
    {
        public int idCursa { get; }
        public int nrLoc { get; }
        public string client { get; }

        public Rezervare(int id, int idCursa, int nrLoc, string client) : base(id)
        {
            this.idCursa = idCursa;
            this.nrLoc = nrLoc;
            this.client = client;
        }
    }
}

File: Semester 4/Advanced Programming/RestAPI/ApplicationModel/src/main/java/app/model/Rezervare.java
package app.model;

import java.io.Serializable;

public class Rezervare implements Entity<Integer>, Serializable {
    private Integer id;
    private Integer idCursa;
    private Integer nrLoc;
    private String client;
    public Rezervare(){

    }
    public void setId(Integer id){
        this.id = id;
    }
    public Rezervare(Integer integer, Integer idCursa, Integer nrLoc, String client) {
        id = integer;
        this.idCursa = idCursa;
        this.nrLoc = nrLoc;
        this.client = client;
    }

    public Integer getIdCursa() {
        return idCursa;
    }

    public Integer getNrLoc() {
        return nrLoc;
    }

    public String getClient() {
        return client;
    }

    public void setIdCursa(Integer idCursa) {
        this.idCursa = idCursa;
    }

    public void setNrLoc(Integer nrLoc) {
        this.nrLoc = nrLoc;
    }

    public void setClient(String client) {
        this.client = client;
    }

    @Override
    public Integer getId() {
        return id;
    }
}
